ci_feature/main.sh

```diff
@@ -196,16 +174,94 @@ if [ "$cAdvisorIsSecure" = true ] ; then
       echo "export IS_SECURE_CADVISOR_PORT=true" >> ~/.bashrc
       export CADVISOR_METRICS_URL="https://$NODE_IP:10250/metrics"
       echo "export CADVISOR_METRICS_URL=https://$NODE_IP:10250/metrics" >> ~/.bashrc
+      echo "Making wget request to cadvisor endpoint /pods with port 10250 to get the configured container runtime on kubelet"
+      podsResponse=$(wget -O- --server-response https://$NODE_IP:10250/pods --no-check-certificate --header="Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)")
```

Comment:
This is not correct way to do this. Because even non-200 response codes will have body and you are relying on body. So it will never go to else part.

Reply:
thanks for pointing this. pushing the fix to handle based on the response code.

Reply:
fixed.